app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -123,6 +129,24 @@ def process_collection(collection, key)
         end
       end
 
+      def gather_data_for_this_region(arm_service)
+        results = []
+        @data[:resource_groups].each do |resource_group|
+          results << arm_service.list(resource_group[:name])
```

Comment:
Yeah, we shouldn't hold this up for it ... but I can see that being a performance drain eventually ... and honestly if the Azure API can't support it, it doesn't make sense to move this logic into the gem ... it's just moving the problem IMO